common/src/main/java/net/william278/huskhomes/command/RtpCommand.java

```diff
@@ -44,7 +41,7 @@ public class RtpCommand extends Command implements UserListTabCompletable {
     protected RtpCommand(@NotNull HuskHomes plugin) {
         super(
                 List.of("rtp"),
-                "[player] [world]",
+                "[player] [world/server] [server (optional)]",
```

Comment:
little nit-pick but `[]` already means its optional so no need for the extra clarification.

Reply:
hmm, so [player] [world/server] [server] is fine?

Reply:
should be
`[player] [<world> [server]/<server>]`

Reply:
Thank you